packages/launch/src/hooks/use-domain-suggestion.ts

```diff
@@ -17,7 +17,7 @@ export function useDomainSuggestion() {
 			if ( ! domainSearch || domainSearch.length < 2 ) {
 				return;
 			}
-			return select( DOMAINS_STORE ).getDomainSuggestions( domainSearch, {
+			return select( DOMAIN_SUGGESTIONS_STORE ).getDomainSuggestions( domainSearch, {
```

Comment:
I see the `DOMAIN_SUGGESTIONS_STORE` being used more in other places even though `DOMAINS_STORE` is shorter.